bedrock/firefox/templates/firefox/ios.html

```diff
@@ -122,6 +144,36 @@
   </div>{#-- /.container --#}
 </section>{#-- /#sync --#}
 
+{% if l10n_has_tag('tracking_protection') and waffle.switch('firefox-ios-global') %}
+<section class="section section-privacy" id="privacy">
+  <div class="container">
+    <h2 class="heading">{{ _('Keep your private info private') }}</h2>
+
+    <p class="lead">
+      {{ _('We believe your personal information belongs to you.') }}
+      {{ _('Firefox for iOS is filled with features to help keep it that way. ') }}
```

Comment:
I shouldn't be allowed to read code at 7AM: it works because we [strip the line](https://github.com/mozilla/bedrock/blob/master/lib/l10n_utils/dotlang.py#L61) before we try to figure out what it is (which also makes the following two `.strip()` on comments and translations not really useful).